setup.cfg

```diff
@@ -76,7 +76,7 @@ install_requires =
     tifffile>=2020.2.16
     toolz>=0.10.0
     tqdm>=4.56.0
-    typing_extensions
+    typing_extensions>=4.2.0
```

Comment:
why is this change required in addition to the constraints?

Reply:
Because withot this in min req test the latest typing extension is isnstaled. And old pydantic is incompatible with latest typing extension

Reply:
I could split it to separate PR, but I want to reduce the time to merge bugfix as fast as possible.